kernel/include/kswap.h

```diff
@@ -235,30 +235,6 @@ static inline void z_swap_unlocked(void)
  *
  * The memory of the dummy thread can be completely uninitialized.
  */
-static inline void z_dummy_thread_init(struct k_thread *dummy_thread)
```

Comment:
Would it make sense to factor this change out to a separate PR?

Reply:
WTF? This was supposed to be part of PR #83871 which shows as being merged. Yet the main branch does not have it.

Reply:
Uh-oh